src/main/java/echoquery/speechAssets/SampleUtterances.txt

```diff
@@ -1,6 +1,20 @@
-TableAggregateIntent how many {table_name} are there
-TableAggregateIntent how many {table_name} are there with {column_name} as {col_value}
-TableAggregateIntent how many {table_name} are there where {column_name} is {col_value}
+CountIntent how many {big sales|table} are there
```

Comment:
the reason I have big sales instead of sales is so Alexa will listen to two-word table names as well as one word table names.